Makefile

```diff
@@ -1,4 +1,6 @@
 export GOPROXY=https://proxy.golang.org
+export GOTOOLCHAIN=local
+export GO111MODULE=on
```

Comment:
(Maybe not in this PR?) I think with current Go versions (the `go 1.21` directive in `go.mod` is being enforced, so we _can_ assume a recent version), it should be possible to never refer to this variable, and to assume modules throughout.

Reply:
“this variable” being `GO111MODULE`, to be explicit.